index/extract_refs.go

```diff
@@ -332,9 +331,12 @@ func (index *SpecIndex) ExtractRefs(node, parent *yaml.Node, seenPath []string,
 					if len(seenPath) > 0 {
 						lastItem := seenPath[len(seenPath)-1]
 						if lastItem == "properties" {
+							seenPath = append(seenPath, n.Value)
```

Comment:
@daveshanley also contains a fix for enum extraction that sorts the issues we were seeing after the last change

Reply:
fixes https://github.com/pb33f/libopenapi/issues/175